### ISSUE TITLE
validation set?

### ISSUE THREAD

**Issue by hzhz2020** [salesforce/CoMatch#9]
Dear Author 
Thank you for this exciting work!
I have a clarification question regarding the experiment setting: Did you use any validation data to tune the hyperparameters for CoMatch? How did u choose the alpha, lambda etc?
